ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/common/BytemanControlledXAResource.java

```diff
@@ -0,0 +1,224 @@
+package com.hp.mwtests.ts.jta.common;
```

Comment:
Hmm, I think it is missing a copyright header. I think it's possible all the new files here are. Please can I ask you to check  if we put the copyright headers on btm files and if so add it to those, but for the Java ones I would presume we do so please may I ask for that?

Reply:
Added it. Thanks Tom

Reply:
Thank you